common/changes/office-ui-fabric-react/documentcard-basic-example-href-change_2019-01-14-18-37.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Changed href in basic example to bing.com",
+      "type": "patch"
```

Also, the type here should be `none` as this is an example change that should not trigger a release. @micahgodbolt can you confirm on my suggestion?
```suggestion
      "type": "none"
```